client/src/app/site/tags/components/tag-list.component.html

```diff
@@ -0,0 +1,43 @@
+<os-head-bar appName="Tags" plusButton=true (plusButtonClicked)=onPlusButton() (ellipsisMenuItem)=onEllipsisItem($event)>
+</os-head-bar>
+<div class="on-transition-fade new-tag-form" *ngIf="tagNewEdit">
+    <form #newTagForm="ngForm" (ngSubmit)="saveTag(newTagForm.form, $event)" (keydown)="keyDownFunction($event, newTagForm.form)">
```

Comment:
(no change required)
fyi: We usually use FormGroup and FormControls for easier access and form validation. The two forms could easily be unified.